### PR TITLE
Render postprocessing

### PR DESCRIPTION
**Pull request by awhiemstra**
The Cura-related render postprocessing work.

### CODE REVIEW

File: plugins/SolidView/SolidView.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Uranium is released under the terms of the AGPLv3 or higher.
```
Comment (nallath) [inline plugins/SolidView/SolidView.py]: Cura :)

File: plugins/SolidView/SolidView.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+from UM.Application import Application
+from UM.Math.Color import Color
+from UM.Preferences import Preferences
+from UM.View.Renderer import Renderer
+
+from UM.View.GL.OpenGL import OpenGL
+
+import math
+
+## Standard view for mesh models.
```
Comment (nallath) [inline plugins/SolidView/SolidView.py]: How is this different from the simple view? I find solid view as a name a bit confusing (as the models displayed are not solid by definition). Solid models would be the stuff that open-scad generates or some voxel implementation.

File: plugins/SolidView/__init__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2015 Ultimaker B.V.
+# Uranium is released under the terms of the AGPLv3 or higher.
```
Comment (nallath) [inline plugins/SolidView/__init__.py]: Cura

File: plugins/SolidView/SolidView.py (hunk 2 above)
Comment (awhiemstra) [inline plugins/SolidView/SolidView.py]: It is different in that it also does the checking if things are inside the build volume and displays overhang. The name "SolidView" comes from the label used it in the view switching dropdown, "Solid". Whether or not that makes sense is up for discussion.

**Comment by awhiemstra** [comment]
This has now been merged.

**Comment by daid** [comment]
Small notes, after this merge, I get an extra view mode called "Simple".
And I did this small change to improve error messages on shader errors:
https://github.com/Ultimaker/Uranium/commit/edefc178e6bed7d2b8a961e7a4b8bffe93c69478
(Was running into a shader compile error due to local edits, but the message did not really help)

**Pull request closed without merging**